apps/examples/testcase/le_tc/network/itc_net_connect.c

```diff
@@ -51,7 +51,7 @@ static void *server_connect(void *ptr_num_clients)
 	struct sockaddr_in address;
 	int addrlen = sizeof(address);
 	int num_clients = *((int *)ptr_num_clients);
-	char buffer[BUFF_LEN] = {0};
+	char buffer[BUFF_LEN + 1] = {0};
```

Comment:
That is quite true. So I removed 'BUFF_LEN+1' and 'strlen(buffer) + 1' following your suggestion, but without changing BUFF_LEN itself, because 'buffer' has two sentences separated by NULL in the middle. That means strlen(buffer) returns the length of first sentences, which causes easy-misunderstanding.

Modifications completed. Thanks for the comment!